collect_app/src/main/java/org/odk/collect/android/utilities/DateTimeUtils.java

```diff
@@ -45,13 +45,13 @@ private static String getGregorianDateTimeLabel(Date date, DatePickerDetails dat
             String format = android.text.format.DateFormat.getBestDateTimePattern(locale, getDateTimePattern(containsTime, datePickerDetails));
             dateFormatter = new SimpleDateFormat(format, locale);
         } else {
-            dateFormatter = DateFormat.getDateTimeInstance(DateFormat.DEFAULT, DateFormat.DEFAULT, locale);
+            dateFormatter = DateFormat.getDateTimeInstance(DateFormat.LONG, DateFormat.LONG, locale);
```

Comment:
This looks like it will change the format (for Android versions below Jelly Bean) from "Nov 15, 2018" (`DEFAULT` is equivalent to `MEDIUM`) to "November 15, 2018".  Why do we need to do that?

Reply:
> Currently, the date that is displayed in English for all languages is displayed as November 15, 2018

I don't see this as the default behavior in the `master` branch. It is in format "Nov 15, 2018".